### DISCUSSION TITLE
Thai Support

### DISCUSSION THREAD

**Discussion by korakot** [deepgram/community#455] [Feature Requests]
Any plan to support Thai language?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment]
Hi @korakot, we don't have immediate plans to add Deepgram model support for Thai. We do offer Thai through our managed Whisper for pre-recorded audio only (not streaming). You can submit your transcription requests with the parameters `model=whisper&language=th`.
